packages/@aws-cdk/aws-dynamodb/lib/table-time-to-live-provider.ts

```diff
@@ -0,0 +1,67 @@
+import * as path from 'path';
+import * as lambda from '@aws-cdk/aws-lambda';
+import { Duration, Stack } from '@aws-cdk/core';
+import * as cr from '@aws-cdk/custom-resources';
+import { Construct } from 'constructs';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct as CoreConstruct } from '@aws-cdk/core';
+
+export class TableTimeToLiveProvider extends CoreConstruct {
```

Comment:
Just `TimeToLiveProvider` maybe?

Reply:
If you think that fits better with the style of this project, I can change it to `TimeToLiveProvider`. I have gone ahead and updated it.